src/tbb/arena_slot.cpp

```diff
@@ -59,6 +59,9 @@ d1::task* arena_slot::get_task_impl(size_t T, execution_data_ext& ed, bool& task
 }
 
 d1::task* arena_slot::get_task(execution_data_ext& ed, isolation_type isolation) {
+#ifdef __TBB_STATISTICS
+    governor::get_thread_data()->add_statistics(thread_data::Statistics::GET_TASK);
+#endif /* __TBB_STATISTICS */
```

Comment:
What about to wrap this three lines into one, e.g. marco or function (that is empty if __TBB_STATISTICS is not set)?